processing-service-core/tenant-config-options.md

```diff
@@ -0,0 +1,200 @@
+# Overview of tenant specific config options
+
+## Setting tenant configurations
+
+A tenant can specify custom values for configuration that can then be used when creating their workflow for the enrichment process.
+A tenant can only specify a custom configuration value for a configuration that already has a global configuration option, if a global configuration option does not exist for the configuration that the tenant tries to set the service will return an APIException to make the user aware that a global configuration must be created for first.
+If a tenant does not have a custom configuration set for the key provided in this call then one will be created for the tenant, if one does exist then its value will be updated with the value provided.
+
+There are two ways that a tenants custom configuration can be created, either per configuration, or as a bulk create request.
+
+### Per config creation:
+
+To create a config for a user the api requires several pieces of information to create the configuration and store them for the tenant, these are:
```

Comment:
I'm not sure about having this level of detail here where you cover the parameters and paths to create the configs. These API paths and parameters are documented in the contract for this project and we don't want to have multiple places in the same project that require updating if we change that. This reads more like documentation for the github io site as a Getting Started guide to using the config options (those docs are in this repo but they are separate from the implementation projects and tend to be updated at a later point once the implementation is fully settled down before release).

I think this should be describing what a tenant config is (already done in your 'Setting tenant configurations' section, including the line about it being possible to perform operations in bulk) and what an effective config is (done under your 'Get Effective Config' section).